### PR TITLE
Replaced former icon with a SVG code + implement B,  R, O icons

### PR DESCRIPTION
**Pull request by elRomano**
Issue:
Adding the right icons
## Changes
- Removed styling
- added consts with SVG code
- used a switch for the specific part of the icons

### CODE REVIEW

**Comment by elRomano** [comment]
@BenoitSerrano

File: packages/generic/client/src/components/business/PublicationCategoryBadge.tsx [hunk 1]
```diff
@@ -5,33 +5,44 @@ import { Text } from '../generic';
...
-    },
-    badgeText: {
-      paddingTop: '4px',
-    },
-  };
+function getIcon(publicationCategoryLetter: string) {
+  switch (publicationCategoryLetter) {
+    case 'B':
+      return ICON_B;
+      break;
```
Comment (BenoitSerrano) [inline packages/generic/client/src/components/business/PublicationCategoryBadge.tsx:37]: Pas besoin du break si tu fais un return

File: packages/generic/client/src/components/business/PublicationCategoryBadge.tsx [hunk 2]
```diff
@@ -5,33 +5,44 @@ import { Text } from '../generic';
...
 function PublicationCategoryBadge(props: { publicationCategoryLetter: string }) {
-  const theme = useCustomTheme();
-  const styles = buildStyles(theme);
   return (
-    <div style={styles.badge}>
-      <Text variant="h3" style={styles.badgeText}>
-        {props.publicationCategoryLetter}
-      </Text>
-    </div>
+    // <div style={styles.badge}>
```
Comment (BenoitSerrano) [inline packages/generic/client/src/components/business/PublicationCategoryBadge.tsx:19]: supprime le comment :)

**Comment by elRomano** [comment]
@BenoitSerrano

**Review by fondation451 — APPROVED** [review]

**Pull request merged**